src/styles/index.scss

```diff
@@ -1,3 +1,8 @@
+@import '/src/styles/blocks/variables';
+@import '/src/styles/blocks/stars';
+@import '/src/styles/blocks/card';
+@import '/src/styles/blocks/header';
```

Comment:
Verify that all import paths are correct and that the corresponding SCSS files exist at the specified locations. Incorrect paths will result in compilation errors.